### ISSUE TITLE
CKEditor: select/upload image dialog has inconsistent styling of fields.

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#3483]
<img width="777" alt="screen shot 2019-01-16 at 5 21 21 pm" src="https://user-images.githubusercontent.com/2423362/51259089-1f36bf00-19b4-11e9-95aa-a214636f9c58.png">

Seems like in order to match the style of the rest of the text fields throughout the admin UI, the "choose file" field should be a few pixels shorter, and the rest of the fields (alt text, image width/height) should be a few pixels taller.

**Comment by herbdool** [comment]
Marking it as a bug is controversial, perhaps we need a new UX tag. This isn't really a bug.

**Comment by klonos** [comment]
I understand where you're coming form @herbdool, but seeing it from a theme perspective, different style for similar elements is a bug. Right?

**Comment by herbdool** [comment]
We need to be able to differentiate between things that make some site unusable, and things where users can work around inconsistencies. There's an urgency to certain broken features and it's hard to find which ones those are right now.

**Comment by olafgrabienski** [comment]
> perhaps we need a new UX tag. This isn't really a bug.

I agree. We used to use the `[UX]` prefix in issue titles but it's not obvious as an option, so a *UX* tag seems to be better.
